### ISSUE TITLE
Support simulcast

### ISSUE THREAD

**Issue by sax** [livinginthepast/specter#8]
Is it possible to have multiple peer connections running in one instance of the NIF, and have simulcast work?

Does this require all media packets to bubble out into Elixir, or could this be done with only RTCP feedback packets?

**Comment by sax** [comment]
This is lower priority for me than other things, and is obviously gated on being able to stream tracks. Tracking it as an issue as a place for recording thoughts and relevant links.

See: https://github.com/webrtc-rs/examples/tree/5a0e2861c66a45fca93aadf9e70a5b045b26dc9e/examples/simulcast.

I could see this done in at least two ways:
* Keep everything in Rust, and make it transparent to users of the library
* Receive feedback reports in Elixir, and the minimum set of functions required to change tracks
